lc-hax/Scripts/Modules/TriggerMod.cs

```diff
@@ -75,7 +75,17 @@ public sealed class TriggerMod : MonoBehaviour, IEnemyPrompter {
             }
 
             if (collider.TryGetComponent(out Turret turret)) {
-                turret.EnterBerserkModeServerRpc(-1);
+                if (turret.turretActive) {
+                    if (turret.turretMode != TurretMode.Berserk) {
+                        turret.EnterBerserkModeServerRpc(-1);
+                    }
+                    else {
+                        turret.ToggleTurretServerRpc(false);
+                    }
+                }
+                else {
+                    turret.ToggleTurretServerRpc(true);
+                }
```

Comment:
I don't understand what's going on here. You want it so that we can only turn off the turret AFTER toggling berserk? And why is `stunclick` not sufficient for this.

Reply:
so you can disguise it by hitting it with a shovel and mess with players? lol

Reply:
I removed it as requested and i did not know stunclick did that